src/main/java/forestry/api/apiculture/IBeekeepingMode.java

```diff
@@ -37,11 +37,6 @@
 	 */
 	boolean isFatigued(IBee queen, IBeeHousing housing);
 
-	/**
-	 * @return true if the queen is being overworked in the bee housing (with chance). will trigger a negative effect.
-	 */
-	boolean isOverworked(IBee queen, IBeeHousing housing);
```

Comment:
To keep backward compatibility with mods that might use this method, you should deprecate this method like this:
```java
	/**		
	 * @return true if the queen is being overworked in the bee housing (with chance). will trigger a negative effect.		
     * @deprecated bees max out at 16x productivity instead of being overworked.
     * TODO remove this method in 1.13
	 */		
    @Deprecated
	default boolean isOverworked(IBee queen, IBeeHousing housing) {
        return false;
    }
```

Reply:
Makes sense on retrospect not to break the interface 👍